omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1988 @@
+      - Dropped
+    * - 11.10
+      - from Oct 2011
+      - to May 2013
+      - Dropped
+      - Dropped
+      - Dropped
+    * - 12.04 (.4) LTS
+      - from Apr 2012
+      - to Apr 2017
+      - Recommended
+      - Deprecated
+      - Dropped
+    * - 12.10
+      - from Oct 2012
+      - to May 2014
+      - Dropped
+      - Dropped
+      - Dropped
+    * - 13.04
+      - from Apr 2013
+      - to Jan 2014
+      - Dropped
+      - Dropped
+      - Dropped
+    * - 13.10
+      - from Oct 2013
+      - to Jul 2014
+      - Dropped
+      - Dropped
+      - Dropped
+    * - 14.04 LTS
+      - from Apr 2014
+      - to Apr 2019
+      - Supported
+      - Recommended
+      - Recommended
+    * - 14.10
+      - from Oct 2014
+      - to Jul 2015
+      - Supported
+      - Supported
+      - Supported
+    * - 15.04
+      - TBA
+      - TBA
+      - Unsupported
+      - Unsupported
+      - Upcoming
+
+Only the LTS releases are supported due to resource limitations upon
+CI and testing. Only the last two LTS releases are supported (being a
+bit more frequent than CentOS/RHEL). There is currently no CI testing
+for any version, but some developer use of 12.04 LTS, 14.04 LTS and
+more recent non-LTS releases.
+
+Linux (Debian stable)
+---------------------
+
+`General overview <https://www.debian.org/releases/>`__
+
+.. tabularcolumns:: |l|l|l|l|l|l|J|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - OMERO 5.0
+      - OMERO 5.1
+      - OMERO 5.2
+      - Details
+    * - 5.0
+      - from Feb 2009
+      - to Feb 2012
+      - Dropped
+      - Dropped
+      - Dropped
+      - `Reference <https://www.debian.org/releases/lenny/>`__
+    * - 6.0
+      - from Feb 2011
+      - to Feb 2016
+      - Supported
+      - Deprecated
+      - Dropped
+      - `Reference <https://www.debian.org/releases/squeeze/>`__
+    * - 7.0
+      - from May 2013
+      - TBA
+      - Recommended
+      - Recommended
+      - Deprecated
+      - `Reference <https://www.debian.org/releases/wheezy/>`__
+    * - 8.0
+      - TBA
+      - TBA
+      - Unsupported
+      - Unsupported
+      - Supported
+      - `Reference <https://www.debian.org/releases/jessie/>`__
+
+Stable releases 7.x and 6.x are supported. There is no CI testing for
+any version, and some developer use of 7.x.
+
+Linux (Debian testing and unstable)
+-----------------------------------
+
+Due to the fast pace of Debian developement, it only really makes
+sense to support the last one or two stable releases; the testing and
+unstable requirements will be covered in the dependencies, below.
+
+Hardware support
+================
+
+
+Microsoft Windows
+-----------------
+
+Supports amd64 (64-bit) and i386 / x86 (32-bit) for all supported
+versions; most systems are today running x64 with most hardware being
+x64, but there is still a niche of x86 users, estimated <25% and
+falling globally (OMERO usage stats here?)
```

Comment:
Do we have these stats immediately to hand? If not, can you just remove reference to them here and we'll add them later.